src/DocumentFormat.OpenXml/SimpleTypes/OpenXmlComparableSimpleReference.cs

```diff
@@ -94,26 +94,28 @@ public override bool Equals(object? obj)
             {
                 return false;
             }
-            else if (Value is null)
+
+            if (Value is null)
```

Comment:
why remove the else?

Reply:
It's not required (see the `return false` above).